### PR TITLE
[Term Entry] Java Date Method: .after()

### PR DESCRIPTION
**Pull request by frhadjmli**
### Description

- Add `Date.after()` term for Java
- closes issue #2685 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/date/terms/after/after.md
Comment (dakshdeepHERE) [inline content/java/concepts/date/terms/after/after.md:16]:
```suggestion
The **`Date.after()`** method checks whether the first date occurs after the second date and returns a boolean value.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
A change from my side

File: content/java/concepts/date/terms/after/after.md
Comment (SSwiniarski) [inline content/java/concepts/date/terms/after/after.md:3]:
```suggestion
Description: 'Checks if a date occurs after a specified date.'
```

File: content/java/concepts/date/terms/after/after.md
Comment (SSwiniarski) [inline content/java/concepts/date/terms/after/after.md:16]:
```suggestion
The **`Date.after()`** method checks whether a date occurs after a specified date and returns a boolean value.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@frhadjmli I did a first maintainer review and left a couple of comments.

**Comment by frhadjmli** [comment]
@SSwiniarski I made the changes as requested.

**Review by SSwiniarski — APPROVED** [review]
@frhadjmli Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@frhadjmli @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@frhadjmli Congrats on the new entry! Live link here:https://www.codecademy.com/resources/docs/java/date/after

image

**Pull request merged**
